src/js/components/DataTableColumns/stories/Pinned.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {
+  Box,
+  Data,
+  DataSearch,
+  DataSummary,
+  DataTable,
+  DataTableColumns,
+  Toolbar,
+} from 'grommet';
+import { columns, DATA } from '../../DataTable/stories/data';
+
+// simplify option label for name property
+const options = columns.map(({ header, property }, i) => ({
+  property,
+  label: property === 'name' ? 'Name' : header,
+  pinned: i === 0 || i === 2,
```

Comment:
```suggestion
  pinned: i === 0 || i === 1,
```
Tiny suggestion to be a bit more aligned with common use cases.